ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -45,7 +46,8 @@ @implementation ADAuthenticationRequest (WebRequest)
 - (void)executeRequest:(NSDictionary *)request_data
             completion:(ADAuthenticationCallback)completionBlock
 {
-    NSString* urlString = [_context.authority stringByAppendingString:OAUTH2_TOKEN_SUFFIX];
+    NSString *authority = [NSString adIsStringNilOrBlank:_cloudAuthority] ? _context.authority : _cloudAuthority;
```

Comment:
It's tempting, if we're forcing developers to be aware of this anyways for sovereign aware applications, to always cache using the home tenant that comes back instead of common.